modules/tinymce/src/themes/silver/test/ts/webdriver/dialogs/IFrameDialogTest.ts

```diff
@@ -13,7 +13,7 @@ import * as WindowManager from 'tinymce/themes/silver/ui/dialog/WindowManager';
 import * as TestExtras from '../../module/TestExtras';
 
 describe('webdriver.tinymce.themes.silver.dialogs.IFrameDialogTest', () => {
-  const isFirefox = PlatformDetection.detect().browser.isFirefox();
+  const isFirefoxBelove126 = PlatformDetection.detect().browser.isFirefox() && PlatformDetection.detect().browser.version.major < 126;
```

Comment:
Maybe add a comment that we can remove this once lambdatest updates all their browsers.

Reply:
done

Reply:
Should be below not belove.

Reply:
fixed, I thought I already fixed it but probably I pressed `ctrl+z` instead of `ctrl+s` :sweat_smile: